src/uv_append.c

```diff
@@ -622,6 +622,24 @@ static int uvAppendEnqueueRequest(struct uv *uv, struct uvAppend *append)
     return rv;
 }
 
+/* Ensure all entry buffers are 8-byte aligned */
+static int uvEnsureEntryBuffersAligned(struct uv *uv,
```

Comment:
Nit: I'd rename this to `uvCheckEntryBuffersAligned`. Throughout the code base "ensure" usually means things like "see if this buffer is at least N bytes, and enlarge it if it's not" (i.e. "make sure things are in a certain way"), while "check" usually means "return an error if something is not in a certain way".